src/js/chapter-factory.js

```diff
@@ -0,0 +1,56 @@
+import { makeDraggable } from "./drag-drop";
+
+const CreateChapter = (text, img) => {
```

Comment:
I am wondering if it makes maybe sense to have a chapter object as input parameter with the properties: dragImages, dropImage, text, question, completed, ...

So make this more into a render-chapter-module and have another module with a factory for creating chapter objects.

Reply:
I'm not sure if i understand correctly, but do you mean that we should write factory for creating objects such as text, images etc. and then we should have separate function just to create DOM structure in which we will append all those elements?

Reply:
I mean having one factory that creates chapter objects which have the properties text, images, ect. (without any rendering functions) - I think such chapter objects could not only be used by a chapter-rendering module but also e.g. by the game-logic module. In many projects in the Odin curriculum we were encouraged to create factories which produce objects nothing having to do with rendering. And then do the rendering separately. One example is the todo list project.

Reply:
OK, now i understand. I like this approach and if the rest is fine with that, i'll write factory for creating chapter objects. @Tudwall @emdevs let me know what you think.

Reply:
I'm fine with it too! Gonna modify my starting page module so it doesn't render then :)